src/components/manage/Blocks/Listing/ListingBody.jsx

```diff
@@ -68,9 +87,11 @@ const ListingBody = ({ data, properties, intl, path, isEditMode }) => {
     );
   }
 
+  console.log(listingItems);
```

Comment:
console.log

Reply:
@tiberiuichim I recall that I added an eslint rule to report console.log statements. No idea why this does not kick in...